### ISSUE TITLE
Adding new table record with Date column crashes

### ISSUE THREAD

**Issue by neilworley** [thedevdojo/voyager#1451]
- Laravel Version: 5.4.28
- Voyager Version: 0.11.*
- PHP Version: 5.6.30
- Database Driver & Version: mySQL 10.1.21-MariaDB

### Description:
When a field is defined as Date it is trying to save as date and time.

### Steps To Reproduce:
Add a field which has Date as its Input Type. When you go to add a new record the selector is a calendar and it returns the time as well. When you click save you get an error. 

```
(3/3)
                                        QueryException
                                    
                                    SQLSTATE[22007]:
 Invalid datetime format: 1292 Incorrect date value: '07/08/2017 9:43 
PM' for column 'date_of_birth' at row 1 (SQL: insert into `rabbits` 
(`name`, `owner_id`, `date_of_birth`, `pp_easy`, `pp_medium`, 
`pp_difficult`, `long_jump`, `high_jump`, `image`, `updated_at`, 
`created_at`) values (Test, 1, 07/08/2017 9:43 PM, 0, 0, 0, 0, 0, , 
2017-07-08 21:44:29, 2017-07-08 21:44:29))
--
in Connection.php (line 647)
at Connection->runQueryCallback('insert
 into `rabbits` (`name`, `owner_id`, `date_of_birth`, `pp_easy`, 
`pp_medium`, `pp_difficult`, `long_jump`, `high_jump`, `image`, 
`updated_at`, `created_at`) values (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)', array('Test', '1', '07/08/2017 9:43 PM', '0', '0', '0', '0', '0', null, '2017-07-08 21:44:29', '2017-07-08 21:44:29'), object(Closure))in Connection.php (line 607)
at Connection->run('insert
 into `rabbits` (`name`, `owner_id`, `date_of_birth`, `pp_easy`, 
`pp_medium`, `pp_difficult`, `long_jump`, `high_jump`, `image`, 
`updated_at`, `created_at`) values (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)', array('Test', '1', '07/08/2017 9:43 PM', '0', '0', '0', '0', '0', null, '2017-07-08 21:44:29', '2017-07-08 21:44:29'), object(Closure))in Connection.php (line 450)
at Connection->statement('insert
 into `rabbits` (`name`, `owner_id`, `date_of_birth`, `pp_easy`, 
`pp_medium`, `pp_difficult`, `long_jump`, `high_jump`, `image`, 
`updated_at`, `created_at`) values (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)', array('Test', '1', '07/08/2017 9:43 PM', '0', '0', '0', '0', '0', null, '2017-07-08 21:44:29', '2017-07-08 21:44:29'))in Connection.php (line 404)
at Connection->insert('insert
 into `rabbits` (`name`, `owner_id`, `date_of_birth`, `pp_easy`, 
`pp_medium`, `pp_difficult`, `long_jump`, `high_jump`, `image`, 
`updated_at`, `created_at`) values (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)', array('Test', '1', '07/08/2017 9:43 PM', '0', '0', '0', '0', '0', null, '2017-07-08 21:44:29', '2017-07-08 21:44:29'))in Processor.php (line 32)
at Processor->processInsertGetId(object(Builder),
 'insert into `rabbits` (`name`, `owner_id`, `date_of_birth`, `pp_easy`,
 `pp_medium`, `pp_difficult`, `long_jump`, `high_jump`, `image`, 
`updated_at`, `created_at`) values (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)', array('Test', '1', '07/08/2017 9:43 PM', '0', '0', '0', '0', '0', null, '2017-07-08 21:44:29', '2017-07-08 21:44:29'), 'id')in Builder.php (line 2138)
at Builder->insertGetId(array('name'
 => 'Test', 'owner_id' => '1', 'date_of_birth' => '07/08/2017 
9:43 PM', 'pp_easy' => '0', 'pp_medium' => '0', 'pp_difficult' 
=> '0', 'long_jump' => '0', 'high_jump' => '0', 'image' => null, 'updated_at' => '2017-07-08 21:44:29', 'created_at' => '2017-07-08 21:44:29'), 'id')in Builder.php (line 1251)
at Builder->__call('insertGetId', array(array('name'
 => 'Test', 'owner_id' => '1', 'date_of_birth' => '07/08/2017 
9:43 PM', 'pp_easy' => '0', 'pp_medium' => '0', 'pp_difficult' 
=> '0', 'long_jump' => '0', 'high_jump' => '0', 'image' => null, 'updated_at' => '2017-07-08 21:44:29', 'created_at' => '2017-07-08 21:44:29'), 'id'))in Model.php (line 684)
at Builder->insertGetId(array('name'
 => 'Test', 'owner_id' => '1', 'date_of_birth' => '07/08/2017 
9:43 PM', 'pp_easy' => '0', 'pp_medium' => '0', 'pp_difficult' 
=> '0', 'long_jump' => '0', 'high_jump' => '0', 'image' => null, 'updated_at' => '2017-07-08 21:44:29', 'created_at' => '2017-07-08 21:44:29'), 'id')in Model.php (line 684)
at Model->insertAndSetId(object(Builder), array('name'
 => 'Test', 'owner_id' => '1', 'date_of_birth' => '07/08/2017 
9:43 PM', 'pp_easy' => '0', 'pp_medium' => '0', 'pp_difficult' 
=> '0', 'long_jump' => '0', 'high_jump' => '0', 'image' => null, 'updated_at' => '2017-07-08 21:44:29', 'created_at' => '2017-07-08 21:44:29'))in Model.php (line 649)
at Model->performInsert(object(Builder))in Model.php (line 518)
at Model->save()in Controller.php (line 81)
at Controller->insertUpdateData(object(Request), 'rabbits', object(Collection), object(Rabbit))in VoyagerBreadController.php (line 244)
at VoyagerBreadController->store(object(Request))
```

I added data using phpmyadmin like so:
```
INSERT INTO `rabbits` (`id`, `name`, `owner_id`, `date_of_birth`, `pp_easy`, `pp_medium`, `pp_difficult`, `long_jump`, `high_jump`, `image`, `created_at`, `updated_at`) VALUES (NULL, 'Test', '1', '2013-07-05', '0', '0', '0', '1.40', '0.63', '', '2017-07-08 20:41:39', NULL);
```
and it worked.

**Comment by lancepioch** [comment]
Labels: bug

**Comment by handiwijoyo** [comment]
Can someone confirm this has been fixed in `1.0` ?

**Comment by jannejava** [comment]
This seems not to be fixed. If a field is defined as date and you are using Safari or Firefox which does not have the date-input field, this is an issue. Bootstrap Datetime Picker uses the English locale and tries to write this format into the field. 

A solution should be to use Carbon::parse before saving the date.
